### DISCUSSION TITLE
Tested hardware, newer models of the XTRA series

### DISCUSSION THREAD

**Discussion by grilli303** [chickey/RS485-WiFi-EPEver#16] [Q&A]
Hi there, I'm very interested in this project.
I plan to get one of the follow-up models of the tracer series, which is called xtra. In my case this would be the XTRA3210N.
Is there any experience as to whether the RS-485 communication also works with these devices?

Kind regards, Daniel

**Comment by Johnhi30** [comment]
Discord comment below looks hopeful .. Cheers John

 Paul.S
 — 
11/04/2021 8:38 AM
Just got the adapter attached to an XTRA2210N-XDS2.  So far so good.  This device is brilliant.  I still have to put the XTRA into the solar system to verifiy all the data.  MQTT is working  too.  Thanks so much Colin.
